src/staticModels/EditorModel.ts

```diff
@@ -20,7 +20,7 @@ export class EditorModel {
     public static viewPortHelper: ViewPortHelper;
 
     public static isLoading: boolean = false;
-    public static viewPortActionsDisabled: boolean = false;
+    public static viewPortActionsDisabled: boolean;
```

Comment:
One more default false that was removed here.

Reply:
![image](https://user-images.githubusercontent.com/28984914/180602903-5aea4ac0-4974-468a-a5bb-04e5946f0d7e.png)
it's already changed, I don't know why it still presist. maybe the diff wasn't latest.

Reply:
`public static viewPortActionsDisabled` was also `false` before your changes.

Reply:
Reversed.